identity/backend/src/main/java/io/camunda/identity/usermanagement/service/UsersService.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.identity.usermanagement.service;
+
+import io.camunda.identity.usermanagement.User;
+import java.util.List;
+import java.util.Optional;
+import org.springframework.dao.DuplicateKeyException;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.core.userdetails.UsernameNotFoundException;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.stereotype.Service;
+
+@Service
+public class UsersService {
+  private final IdentityLocalUserDetailsManager userDetailsManager;
+
+  private final PasswordEncoder passwordEncoder;
+
+  public UsersService(
+      final IdentityLocalUserDetailsManager userDetailsManager,
+      final PasswordEncoder passwordEncoder) {
+    this.userDetailsManager = userDetailsManager;
+    this.passwordEncoder = passwordEncoder;
+  }
+
+  public User createUser(final User user) {
+    try {
+      final UserDetails userDetails =
+          org.springframework.security.core.userdetails.User.withUsername(user.username())
+              .password(user.password())
+              .passwordEncoder(passwordEncoder::encode)
+              .disabled(!user.enabled())
+              .roles("DEFAULT_USER")
```

Comment:
Here I've just tried a default role because it's not possible to create a User without authority in spring. I hope this "DEFAULT_USER it's not be used anywhere for now and later when we start we moving the role/permissions parts to identity we can decide for proper roles. wdyt?